packages/node/test/manual/release-health/session-aggregates/aggregates-disable-single-session.js

```diff
@@ -7,35 +7,33 @@ const { assertSessions, BaseDummyTransport } = require('../test-utils');
 function cleanUpAndExitSuccessfully() {
   console.log('SUCCESS: Session Aggregates payload is correct!');
   server.close();
-  clearInterval(flusher._intervalId)
+  clearInterval(flusher._intervalId);
   process.exit(0);
 }
 
 function assertSessionAggregates(session, expected) {
   // For loop is added here just in the rare occasion that the session count do not land in the same aggregate
   // bucket
-  session.aggregates.forEach(function (_, idx) {
+  session.aggregates.forEach(function(_, idx) {
     delete session.aggregates[idx].started;
     // Session Aggregates keys need to be ordered for JSON.stringify comparison
-    const ordered = Object.keys(session.aggregates[idx]).sort().reduce(
-      (obj, key) => {
+    const ordered = Object.keys(session.aggregates[idx])
+      .sort()
+      .reduce((obj, key) => {
         obj[key] = session.aggregates[idx][key];
         return obj;
-      },
-      {}
-    );
+      }, {});
     session.aggregates[idx] = ordered;
-  })
-  assertSessions(session, expected)
+  });
+  assertSessions(session, expected);
 }
 
 class DummyTransport extends BaseDummyTransport {
   sendSession(session) {
-    console.error('FAIL: Received Single Session which should have been deleted in favor of Session Aggregates!');
```

Comment:
Removing this section means we no longer check if the application mode session was dropped in favor of SessionAggregates mode

Reply:
There is only `sendSession` anymore and we do an assert to check for content, so if a single session reaches this it would fail.